### DISCUSSION TITLE
设置密码后,alistv3 有办法在不隐藏文件夹的情况下.防止别人直接访问目录,但是直连OK吗?

### DISCUSSION THREAD

**Discussion by jsjcjsjc** [AlistGo/alist#3209] [Q&A]
我分享的时候比如链接abc.com/files/test.zip，别人通过链接直接可以访问abc.com/files/看到全部文件了，这个有办法防止吗？
感谢

**Comment by anwen-anyi** [comment]
加密 files文件夹，顺便勾选上应用到子文件夹

**Reply by jsjcjsjc** [reply]
> 加密 files文件夹，顺便勾选上应用到子文件夹

感谢,测试了一下似乎不行哈
